pkg/plugins/dashboards.go

```diff
@@ -97,7 +97,7 @@ func loadPluginDashboard(pluginId, path string) (*models.Dashboard, error) {
 	}
 
 	dashboardFilePath := filepath.Join(plugin.PluginDir, path)
-	reader, err := os.Open(dashboardFilePath)
+	reader, err := os.Open(filepath.Clean(dashboardFilePath))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.